internal/controller/vrg_volrep.go

```diff
@@ -2016,6 +2017,15 @@ func (v *VRGInstance) restorePVsAndPVCsFromS3(result *ctrl.Result) (int, error)
 			continue
 		}
 
+		// If in failover mode and no PVs are found, try next S3 profile
+		if v.instance.Spec.Action == ramendrv1alpha1.VRGActionFailover && pvCount == 0 {
+			v.log.Info("No PVs found in the", " S3 profile: ", s3ProfileName)
```

Comment:
`log.Info` takes keys and values. Just change the line to something similar to the suggested code above.